### PR TITLE
add unionml a new mlops for model serve to reduce boilerplate

### PR DESCRIPTION
**Pull request by kumarivin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Transformer-deploy](https://github.com/ELS-RD/transformer-deploy/) ![](https://img.shields.io/github/stars/ELS-RD/transformer-deploy.svg?style=social) - Transformer-deploy is an efficient, scalable and enterprise-grade CPU/GPU inference server for Hugging Face transformer models.
 * [Triton Inference Server](https://github.com/triton-inference-server/server) ![](https://img.shields.io/github/stars/triton-inference-server/server.svg?style=social) - Triton is a high performance open source serving software to deploy AI models from any framework on GPU & CPU while maximizing utilization.
 * [WhyLogs](https://github.com/whylabs/whylogs-python) ![](https://img.shields.io/github/stars/whylabs/whylogs-python.svg?style=social) - Lightweight solution for profiling and monitoring your ML data pipeline end-to-end
-
+* [UnionML](https://github.com/unionai-oss/unionml) ![](https://img.shields.io/github/stars/unionai-oss/unionml.svg?style=social) - UnionML is an open source MLOps framework that aims to reduce the boilerplate and friction that comes with building models and deploying them to production.
```
Comment (axsaucedo) [inline README.md:205]: Thank you for the addition - only change is to make sure it's in alphabetical order

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Thanks for the contribuion - I've added a comment

File: README.md (hunk 1 above)
Comment (kumarivin) [reply README.md:204]: @axsaucedo updated as advised

File: README.md [hunk 2]
```diff
@@ -201,9 +201,9 @@ This repository contains a curated list of awesome open source libraries that wi
...
 * [Transformer-deploy](https://github.com/ELS-RD/transformer-deploy/) ![](https://img.shields.io/github/stars/ELS-RD/transformer-deploy.svg?style=social) - Transformer-deploy is an efficient, scalable and enterprise-grade CPU/GPU inference server for Hugging Face transformer models.
 * [Triton Inference Server](https://github.com/triton-inference-server/server) ![](https://img.shields.io/github/stars/triton-inference-server/server.svg?style=social) - Triton is a high performance open source serving software to deploy AI models from any framework on GPU & CPU while maximizing utilization.
+* [UnionML](https://github.com/unionai-oss/unionml) ![](https://img.shields.io/github/stars/unionai-oss/unionml.svg?style=social) - UnionML is an open source MLOps framework that aims to reduce the boilerplate and friction that comes with building models and deploying them to production.
 * [WhyLogs](https://github.com/whylabs/whylogs-python) ![](https://img.shields.io/github/stars/whylabs/whylogs-python.svg?style=social) - Lightweight solution for profiling and monitoring your ML data pipeline end-to-end
 
-
```
Comment (zhimin-z) [inline README.md:206]: It seems like a line is missing. @kumarivin

**Pull request merged**